lib-theming/build.gradle.kts

```diff
@@ -25,6 +27,13 @@ dependencies {
     implementation("androidx.recyclerview:recyclerview:$recyclerVersion")
     implementation("androidx.swiperefreshlayout:swiperefreshlayout:1.1.0")
     implementation("androidx.cardview:cardview:1.0.0")
+
+    // Hilt
+    implementation("com.google.dagger:hilt-android:2.28-alpha")
```

Comment:
Should we lift version out as it is used in multiple places?  (Not sure of preferred Droid conventions)

Reply:
yesss I really want to, but I'm trying to see if there's a way to only have to define the entire dependency once rather than in each module because it's already getting tedious